Tests/Integration/Microsoft365DSC/M365DSCIntegration.AAD.Create.Tests.ps1

```diff
@@ -332,6 +332,13 @@
     {
         Master -ConfigurationData $ConfigurationData -Credscredential $Credential
         Start-DscConfiguration Master -Wait -Force -Verbose -ErrorAction Stop
+
+        $TestDscConfigurationResult = Test-DscConfiguration Master -Verbose
+        if ($TestDscConfigurationResult.InDesiredState -ne $true) {
```

Comment:
These files are automatically generated and should never be modified manually

Reply:
Understood - will sort this out